exporter/opentelemetry-exporter-datadog/src/opentelemetry/exporter/datadog/exporter.py

```diff
@@ -282,3 +299,27 @@ def _parse_tags_str(tags_str):
             parsed_tags[key] = value
 
     return parsed_tags
+
+
+def _extract_tags_from_resource(resource):
+    """Parse tags from resource.labels, except service.name which
+    has special significance within datadog"""
+    tags = {}
+    service_name = None
+    if not resource and resource.labels:
+        return [tags, service_name]
+
+    for attribute_key, attribute_value in resource.labels.items():
+        if isinstance(attribute_value, (int, bool, float)):
```

Comment:
We can safely remove this logic since the exporter already makes use of `ddtrace.span.Span.set_tag` for handling `span.attributes`, which does the necessary casting based on the value or special conditions on keys:

https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/span.py#L180-L259

Reply:
makes sense, updated